internal/worker/pool.go

```diff
@@ -0,0 +1,78 @@
+package worker
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/supressionstop/xenking_test_1/internal/config"
+	"github.com/supressionstop/xenking_test_1/internal/usecase"
+)
+
+type Pool struct {
+	workers        []*Worker
+	logger         *slog.Logger
+	FirstSyncChan  chan struct{}
+	workerSyncChan chan struct{}
+	workersErrChan chan error
```

Comment:
nit: It's a bad practice to use type names in variable names. We should always name variables without it's underlying type included.

Reply:
TODO: 
1. remove package or type prefixes;
2. try to find linter.

Reply:
Removed and linters were added.